deployment/ccip/changeset/cs_active_candidate.go

```diff
@@ -111,20 +168,20 @@ func SetCandidatePluginChangeset(
 
 	var (
 		timelocksPerChain = map[uint64]common.Address{
-			homeChainSel: state.Chains[homeChainSel].Timelock.Address(),
+			cfg.HomeChainSelector: state.Chains[cfg.HomeChainSelector].Timelock.Address(),
 		}
 		proposerMCMSes = map[uint64]*gethwrappers.ManyChainMultiSig{
-			homeChainSel: state.Chains[homeChainSel].ProposerMcm,
+			cfg.HomeChainSelector: state.Chains[cfg.HomeChainSelector].ProposerMcm,
 		}
 	)
 	prop, err := proposalutils.BuildProposalFromBatches(
 		timelocksPerChain,
 		proposerMCMSes,
 		[]timelock.BatchChainOperation{{
-			ChainIdentifier: mcms.ChainIdentifier(homeChainSel),
+			ChainIdentifier: mcms.ChainIdentifier(cfg.HomeChainSelector),
 			Batch:           setCandidateMCMSOps,
 		}},
-		"SetCandidate for execution",
+		fmt.Sprintf("SetCandidate for %s plugin", cfg.PluginType.String()),
```

Comment:
Switched up the wording a bit here because we could conceivably setCandidate on either of the plugins at any time